packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -124,6 +124,8 @@ module.exports = (themeOptions = {}) => {
             '.mdx',
             // ".md"
           ],
+          // implement commonmark for stricter compatibility, e.g. backslash transformed to newlines
+          commonmark: true,
```

Comment:
I'm surprised that this works to be honest, as this option is not documented.

https://github.com/gatsbyjs/gatsby/blob/c993d52bff1b2255ba8a2c0d04726bfdb71b070a/packages/gatsby-plugin-mdx/gatsby-node.js#L71-L126

Reply:
<img width="956" alt="image" src="https://user-images.githubusercontent.com/1110551/100351904-7d8b4a80-2fec-11eb-929e-b6035d8c548d.png">

Reply:
I don't think this is the correct approach.

Reply:
Outch, this is a regression of a regression we already had during migration.

Functionally, the fix here is correct, the behavior is one of the relevant differences between "tolerant" and commonmark MD parsing. 

The Gatsby devs have just recently introduced their option validation and their issue tracker shows quite some people finding options that are implemented but not part of their validation.   Their plugins often feed options through to sub-plugins but do not document all of them again - a practice that is annoying in the image related plugins, too. 

PS: The MDX / commonmark topic sounds eeerily similar to the remark upgrade to a pure commonmark parser that we discussed recently @emmenko

Reply:
Yeah, I tried to follow the trail a bit and it seems indeed that the plugin options are passed down to some other libraries, like `remark-parse`.

Alright then, let's keep it like this for now!